### PR TITLE
New package: wmclock-1.0.16

### PR DESCRIPTION
**Pull request by dbear6**

### CODE REVIEW

File: srcpkgs/wmclock/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'wmclock'
+pkgname=wmclock
+version=1.0.16
+revision=1
+wrksrc="dockapps-daaf3aa"
+build_style=gnu-configure
+hostmakedepends="autoconf automake pkg-config"
+makedepends="libX11-devel libXpm-devel libXext-devel"
+depends="libX11 libXpm libXext libxcb libXau libXdmcp"
```
Comment (Vaelatern) [inline srcpkgs/wmclock/template]: Remove this line.

File: srcpkgs/wmclock/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+short_desc="Clock Dockapp for Window Maker"
+maintainer="Daniel James <djames@orcadian.net>"
+license="GPL-2"
+homepage="https://www.dockapps.net/wmclock"
+distfiles="http://www.dockapps.net/download/wmclock-1.0.16.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/wmclock/template]: Substitude `${version}` here.

File: srcpkgs/wmclock/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'wmclock'
+pkgname=wmclock
+version=1.0.16
+revision=1
+wrksrc="dockapps-daaf3aa"
```
Comment (Vaelatern) [inline srcpkgs/wmclock/template]: This a hash at the end? I guess it's okay as left in.

File: srcpkgs/wmclock/template [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+depends="libX11 libXpm libXext libxcb libXau libXdmcp"
+short_desc="Clock Dockapp for Window Maker"
+maintainer="Daniel James <djames@orcadian.net>"
+license="GPL-2"
+homepage="https://www.dockapps.net/wmclock"
+distfiles="http://www.dockapps.net/download/wmclock-1.0.16.tar.gz"
+checksum=3f8bb5df3160d5919d19cf26d45449e6e1260d2a2f485e61f35e64a49639a2d3
+
+pre_configure() {
+        autoreconf -i
```
Comment (Vaelatern) [inline srcpkgs/wmclock/template]: Please see the reason Travis failed. Hint: the failure is from xlint and is on this line.

File: srcpkgs/wmclock/template (hunk 3 above)
Comment (dbear6) [reply srcpkgs/wmclock/template]: @Vaelatern when you extract the .tar.gz, this is the directory name. I am not sure why this is the case.

**Comment by dbear6** [comment]
@Vaelatern the requested changes have been made.

**Pull request merged**